sdks/python/apache_beam/runners/direct/bundle_factory.py

```diff
@@ -198,3 +200,16 @@ def commit(self, synchronized_processing_time):
     self._committed = True
     self._elements = tuple(self._elements)
     self._synchronized_processing_time = synchronized_processing_time
+
+  def get_min_timestamp(self):
+      """Returns the minimum timestamp across all elements in this bundle, or
+      MAX_TIMESTAMP if this bundle is empty."""
```

Comment:
<!--new_thread; commit:61ae92dcffa98203cbe8f56930802405de2524f0; resolved:0-->
Can you make this one line?  For example:

```
   """Returns max element timestamp in bundle or MAX_TIMESTAMP if empty."""
```